src/continuous-scroll.js

```diff
@@ -0,0 +1,202 @@
+/* global TweenLite, TimelineLite, Linear */
+import React from "react";
+import "gsap";
+
+let tween = null,
+  timeline = null;
+
+const ContinuousScroll = React.createClass({
+  propTypes: {
+    type: React.PropTypes.oneOf(["none", "continuous", "page"]),
+    speed: React.PropTypes.oneOf(["slow", "medium", "fast"]),
+    pause: React.PropTypes.number,
+    height: React.PropTypes.number.isRequired,
+    data: React.PropTypes.array.isRequired,
+    isRefreshing: React.PropTypes.bool,
```

Comment:
@stulees I don't believe that the component should manage the data in its own state. It shouldn't care where the data comes from. Additionally, the Widget is already managing the data in its state, so there isn't a need for the component to manage it too.

We'll need to set the `isRefreshing` prop any time the data is refreshed, so that the component knows to duplicate it. `componentDidUpdate` will fire on initial load and on every subsequent render, hence we need a way to be able to distinguish between an initial render and a subsequent render (i.e. a refresh).